.travis.yml

```diff
@@ -1,12 +1,22 @@
 language: c
-sudo: required
-install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-opam.sh
-script: bash -ex .travis-opam.sh && make gh-pages && bash -ex .coverage.sh
+sudo: false
+services:
+  - docker
+install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-docker.sh
+script: bash -ex ./.travis-docker.sh && make gh-pages
 env:
   global:
   - secure: "BcNzNHmOVQQegY8uI2sOhV5RNpW144M7omUbTIATdxMou690ydk9LJDaWowqyv9JruS6n2ZMu3UO2bf/KySxHG56iZ8mtPtUtp98llWydM37dodGLAK09SX8vD9NqOZ5JK9382Ip83s98DjLrZKdftB4FSt82aOB4erF4hB0ChM="
+  - PINS="nbd:. nbd-lwt-unix:. nbd-tool:."
   matrix:
-  - OCAML_VERSION=4.03
-  - OCAML_VERSION=4.04
+  - PACKAGE=nbd  DISTRO="debian-unstable" OCAML_VERSION=4.03.0
```

Comment:
@mseri sorry, I was actually wrong about the doc upload, it might be possible to do in the container, because we can pass extra environment variables into out through `EXTRA_ENV`: https://github.com/ocaml/ocaml-ci-scripts/blob/master/.travis-docker.sh#L22